### PR TITLE
Storage list for lens

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]
overall good

desc says that cmd lists objects in blobstor. At the same time, it does not access the blobstor, only metabase is read. May be this worth to be clarified

**Comment by roman-khimov** [comment]
> May be this worth to be clarified

Done.

File: cmd/neofs-lens/internal/storage/list.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+func listFunc(cmd *cobra.Command, _ []string) {
+	// other targets can be supported
+	w := cmd.OutOrStderr()
+
+	storage := openEngine(cmd)
+	defer storage.Close()
+
+	var p engine.ListWithCursorPrm
+	p.WithCount(1024)
```
Comment (carpawell) [inline cmd/neofs-lens/internal/storage/list.go:32]: magic? can lose 1025-th object and do not understand it?

**Review by carpawell — COMMENTED** [review]
@roman-khimov, as @cthulhu-rider said that is a meta operation and the lens' `meta` subcommand already has it, so why do we need it? It can be confusing.

File: cmd/neofs-lens/internal/storage/list.go (hunk 1 above)
Comment (roman-khimov) [reply cmd/neofs-lens/internal/storage/list.go:32]: It will list everything. Yeah, magic.

Comment (carpawell) [reply cmd/neofs-lens/internal/storage/list.go:32]: oh, almost endless `for`, did not see it. check context b/w iteration then to make it controllable?

**Review by carpawell — APPROVED** [review]
The work is done and it does what it claims so OK, but I would not add this command as the blobstor is a too complex thing to inspect IMO, and it is not about storage only but about managing and background processes too. Also, it is changed quite often.

**Pull request merged**